Sources/SwifterSwift/CoreLocation/CLLocationExtensions.swift

```diff
@@ -63,6 +63,19 @@ public extension CLLocation {
 
         return (degrees + 360).truncatingRemainder(dividingBy: 360)
     }
+
+    /// SwifterSwift: Check the distance to end is less than or equal to the `distance`.
+    ///
+    /// - Parameters:
+    ///   - location: End location.
+    ///   - distance: Range limit distance.
+    ///   - unitLength: The unit of length. Default value is meters.
+    /// - Returns: `true` if the distance between the start and end is less than or equal to the given `distance`.
+    @available(iOS 10.0, macOS 10.12, tvOS 10.0, watchOS 3.0, *)
```

Comment:
The project's iOS version is already 10.0, so no need for it.
```suggestion
    @available(macOS 10.12, tvOS 10.0, watchOS 3.0, *)
```